src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.Win32.cs

```diff
@@ -64,6 +64,8 @@ private void StartRaisingEvents()
         /// <summary>Stop monitoring the current directory.</summary>
         private void StopRaisingEvents()
         {
+            _enabled = false;
```

Comment:
Mainly consistency across implementations so that we always set it to false first thing in StopRaisingEvents.  But in this case, we also want to make sure it's false even if the the directory handle somehow got invalidated.